src/shared/navgen/nav.cpp

```diff
@@ -943,11 +945,27 @@ void NavmeshGenerator::StartGeneration( class_t species )
 	d_->tw = ( gw + ts - 1 ) / ts;
 	d_->th = ( gh + ts - 1 ) / ts;
 
+	float climb = config_.stepSize;
+	if ( sg_botAutojump.Get() > 0.f )
+	{
+		glm::vec3 mins, maxs;
+		BG_BoundingBox( species, &mins, &maxs, nullptr, nullptr, nullptr );
+		float safety = ( maxs.z - mins.z ) * sg_botAutojump.Get();
```

Comment:
Why is the ratio relative to the player's height? I would have thought making it relative to the jump height makes more sense. Dretch has a higher jumpMagnitude than tyrant.

Reply:
this is a limitation of recast, stepsize  should not be bigger than agent's height. It has to be said that using climb height to generate jumping connections is a hack, but the alternative is to manually create offmesh connections for jumps, which is unrealistic giving the number of navmeshes we'd need to maintain ( number of maps * number of classes, so ~2500 ).